Observer/SaveOrderBeforeSalesModelQuote.php

```diff
@@ -65,9 +65,6 @@ public function execute(Observer $observer)
             if ($extensionAttributes === null) {
                 $extensionAttributes = $this->orderExtensionFactory->create();
             }
-            if ($extensionAttributes->getShippingProtection() === null) {
```

Comment:
Why did you remove this?

Reply:
Since it is an class now, `null` seems more fitting since it isn't an associative array anymore